tinygradient.d.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * TinyGradient 0.4.0
+ * Copyright 2014-2018 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
+ * Licensed under MIT (http://opensource.org/licenses/MIT)
+ */
+
+declare var tinygradient: tinygradient;
+
+type ArcMode = boolean | 'short' | 'long';
+
+interface TinyGradient {
+
+    /**
+     * Return new instance with reversed stops
+     * @return {tinygradient}
+     */
+    reverse(): tinygradient;
+
+    /**
+     * Generate gradient with RGBa interpolation
+     * @param {int} steps
+     * @return {tinycolor[]}
+     */
+    rgb(steps: number): tinycolorInstance[];
+
+    /**
+     * Generate gradient with HSVa interpolation
+     * @param {int} steps
+     * @param {Boolean|String} [mode=false]
+     *    - false to step in clockwise
+     *    - true to step in trigonometric order
+     *    - 'short' to use the shortest way
+     *    - 'long' to use the longest way
+     * @return {tinycolor[]}
+     */
+    hsv(steps: number, mode: ArcMode): tinycolorInstance[];
+
+    /**
+     * Generate CSS3 command (no prefix) for this gradient
+     * @param {String} [mode=linear] - 'linear' or 'radial'
+     * @param {String} [direction] - default is 'to right' or 'ellipse at center'
+     * @return {String}
+     */
+    css(mode?: 'linear' | 'radial', direction?: string): string;
+
+    /**
+     * Returns the color at specific position with RGBa interpolation
+     * @param {float} pos, between 0 and 1
+     * @return {tinycolor}
+     */
+    rgbAt(pos: number): tinycolorInstance;
+
+    /**
+     * Returns the color at specific position with HSVa interpolation
+     * @param {float} pos, between 0 and 1
+     * @return {tinycolor}
+     */
+    hsvAt(pos: number): tinycolorInstance;
+
+}
+
+interface tinygradient {
+    /**
+     * @class tinygradient
+     * @param {*} stops
+     */
+    (stops: TinyGradientSteps[]): TinyGradient;
```

Comment:
can't you use ColorInput type from tinycolor here ?

Reply:
I think that's right. I fixed it.